<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="font">
       <font>
        <family>Courier</family>
       </font>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::NoWrap</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>443</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRepeat"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="CuteEditor.qrc">
     <normaloff>:/新前缀/images/action_new.png</normaloff>:/新前缀/images/action_new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="CuteEditor.qrc">
     <normaloff>:/新前缀/images/action_open.png</normaloff>:/新前缀/images/action_open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="CuteEditor.qrc">
     <normaloff>:/新前缀/images/action_save.png</normaloff>:/新前缀/images/action_save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="CuteEditor.qrc">
     <normaloff>:/新前缀/images/action_save_as.png</normaloff>:/新前缀/images/action_save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save as ...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="CuteEditor.qrc">
     <normaloff>:/新前缀/images/action_exit.png</normaloff>:/新前缀/images/action_exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="CuteEditor.qrc">
     <normaloff>:/新前缀/images/action_undo.png</normaloff>:/新前缀/images/action_undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRepeat">
   <property name="icon">
    <iconset resource="CuteEditor.qrc">
     <normaloff>:/新前缀/images/action_redo.png</normaloff>:/新前缀/images/action_redo.png</iconset>
   </property>
   <property name="text">
    <string>Repeat</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="CuteEditor.qrc">
     <normaloff>:/新前缀/images/action_cut.png</normaloff>:/新前缀/images/action_cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="CuteEditor.qrc">
     <normaloff>:/新前缀/images/action_copy.png</normaloff>:/新前缀/images/action_copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="CuteEditor.qrc">
     <normaloff>:/新前缀/images/action_paste.png</normaloff>:/新前缀/images/action_paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="icon">
    <iconset resource="CuteEditor.qrc">
     <normaloff>:/新前缀/images/action_info.png</normaloff>:/新前缀/images/action_info.png</iconset>
   </property>
   <property name="text">
    <string>Info ...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CuteEditor.qrc"/>
 </resources>
 <connections/>
</ui>
